app/components/chart-footnotes.tsx

```diff
@@ -118,17 +118,22 @@ export const ChartFootnotes = ({
           {configKey && shareUrl && (
             <>
               <Box sx={{ display: "inline", mx: 2 }}>·</Box>
-              <Link
-                sx={{ typography: "caption" }}
+              <Button
+                component="a"
+                variant="text"
                 color="primary"
+                size="small"
+                sx={{ fontWeight: "regular", padding: 0 }}
```

Comment:
Nit: Again, this fontWeight: regular :) Should font-weight regualr be the default for Button with variant "text" ? @AnninaWalker do you have a recommendation ?

Reply:
@ptbrowne Not exactly sure what you mean with variant "text"? Buttons fontweight should be regular yes.

Reply:
The variant "text" is when the button has not outline nor fill : https://mui.com/material-ui/react-button/#basic-button. I had put fontWeight: bold onto the button size small + variant text since in the stepper, the links were bold. But now Bartosz has changed the logic to keep fontWeight: regular as the default, and have fontWeight: bold as overrides for the stepper buttons. Does it sound OK to you ?

Reply:
Yes that sounds ok 👍